Backend/NotificationService/benchmarks/delete_connection_benchmark.cpp

```
@@ -0,0 +1,8 @@
+#include <chrono>
+
+#include "../include/notificationservice/SocketRepository.h"
+
+int main2() {
+    std::cout << "Hello";
+    return 0;k
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If k is a macro then please configure it.

(unknownMacro)